src/java.base/share/classes/java/lang/Process.java

```diff
@@ -149,12 +169,85 @@ public abstract class Process {
     private Charset inputCharset;
     private BufferedReader errorReader;
     private Charset errorCharset;
+    private volatile boolean closed;     // true if close() has been called
 
     /**
      * Default constructor for Process.
      */
     public Process() {}
 
+    /**
+     * Close all writer and reader streams and terminate the process.
```

Comment:
Maybe not in this PR but I think we'll have to take a pass over the Process docs so that it uses consistent terminology.  Right now the class description speaks of "Destroying a process" and defines "destroy" methods to "kill" a process.  The "destroyForcibly" method speaks of the process being "forcibly terminated".  The update to the close method means it is specified to "terminate" a process. It just means different terms in the first sentence of method descriptions.  I think we could make "terminate", "terminate forcibly", and "await termination" work.

Reply:
https://bugs.openjdk.org/browse/JDK-8080772 is an enhancement to improve consistency of terminology around processes. I added a comment on termination there.